apps/website/src/components/settings/CourseListRow.test.tsx

```diff
@@ -51,6 +60,23 @@ describe('CourseListRow', () => {
     roundStatus: 'Active',
   };
 
+  beforeEach(() => {
+    vi.useFakeTimers();
+    vi.setSystemTime(new Date('2024-01-01'));
+
+    // Mock the API call for meet person data
+    vi.mocked(useAxios).mockReturnValue([{
+      data: null,
+      loading: false,
+      error: null,
+    }, () => {}, () => {}] as unknown as ReturnType<typeof useAxios>);
```

Comment:
idk mocking time was causing request to fail so I needed to mock axios as well. This was copied from `CourseDetails.test.tsx`